fastapi_cloudevents/cloudevent_request.py

```diff
@@ -31,7 +31,7 @@ def _best_effort_fix_json_data_payload(event: CloudEvent) -> CloudEvent:
 class CloudEventRequest(Request):
     _settings: CloudEventSettings = CloudEventSettings()
     _json: Any
-    _body: Any
+    _body: bytes
```

Comment:
@autra doesn't it break backwards compatibility?

Reply:
I don't think this change (from `Any` to `bytes`) really breaks anything that wasn't broken before. If I understand correctly, `Any` basically means "don't bother checking" to the type checker. If we change it to `bytes`, we align ourselves with the reality (because the runtime type is `bytes` already, whether we type check or not).

For instance, with this snippet, the type checker is happy, but we still have a failure at runtime:

```python
from typing import Any

body: Any = b'foo'

body.decode("UTF-8")
body.append("foo")
```

If we change the type of `body` to `bytes`, same error, but at type checking time. I don't think failing faster should be considered as a breaking change. Actually, people using type checking tool would arguably want that. What do you think?

Reply:
Done!